bench/openssl/optimized/libcrypto-lib-stack.ll

```diff
@@ -479,22 +479,24 @@ if.then9:                                         ; preds = %if.end5
   br label %if.end22
 
 if.else:                                          ; preds = %if.end5
-  %3 = zext nneg i32 %loc to i64
-  %4 = getelementptr ptr, ptr %2, i64 %3
-  %arrayidx14 = getelementptr ptr, ptr %4, i64 1
+  %add = add nuw nsw i32 %loc, 1
```

Comment:
Regression.

Reply:
This is most likely fixed by https://github.com/llvm/llvm-project/commit/b8df88b41c8a1b4e879b4fd34be3522c9b45e86f.